honeybee/radiance/parameters/_parametersbase.py

```diff
@@ -210,13 +210,23 @@ def _parse_rad_parameters(self, parameters_string):
         if parameters_string is None:
             return rad_par
 
-        # split input string: each part will look like key value (eg ad 1)
-        par_list = parameters_string.split("-")
+        # use re to find the start and end index for each parameter in parameter string
+        indices = [(m.start(0), m.end(0)) for m in re.finditer(r'-[a-zA-Z]{1,6}'\
+                    , parameters_string)]
 
-        for p in par_list:
-            key, sep, value = p.partition(" ")
+        indices = [item for sublist in indices for item in sublist]
+
+        indices.append(len(parameters_string))
+
+        for i in xrange(0,len(indices)-2,2):
+            key_start = indices[i] + 1
+            key_end = indices[i+1]
+            key = parameters_string[key_start:key_end]
+
+            value_start = key_end
+            value_end = indices[i+2]
```

Comment:
A couple of PEP8 comments:
1. You need white space around `+`, `-`, etc.
2. You need white space after commas: `(0,len(indices)-2,2)` should be `(0, len(indices) - 2, 2)`

If you are using Atom you can install the plugins to check them for you. `autopep8` even fixes them for you automatically each time you save your file.

PS: For some reason @sariths calls me the PEP8 police :)